src/frontend-samples/version-compare-sample/VersionCompareWidget.tsx

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { assert } from "@bentley/bentleyjs-core";
+import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
+import { AbstractWidgetProps, StagePanelLocation, StagePanelSection, UiItemsProvider, WidgetState } from "@bentley/ui-abstract";
+import { Select, Spinner, SpinnerSize } from "@bentley/ui-core";
+import { useActiveIModelConnection } from "@bentley/ui-framework";
+import "common/samples-common.scss";
+import * as React from "react";
+import { NamedVersion, VersionCompareApi } from "./VersionCompareApi";
+import "./VersionCompare.scss";
+
+export const VersionCompareWidget: React.FunctionComponent = () => {
+  // State Declarations
+  const iModelConnection = useActiveIModelConnection();
+  const [isRequest, setIsRequest] = React.useState<boolean>(false);
+  const [selectVersion, setVersion] = React.useState<NamedVersion>(VersionCompareApi.namedVersions[1]);
```

Comment:
Nitpick: can the sample change iModels? If so, is it guaranteed namedVersions array's length will always be >= 2?

Reply:
Good catch.  Validating Named Version length now.